#This represents a vector in free space -- currently defined to hold the left and right track velocities
float64 left_torque
float64 right_torque
